app/data_model/answer_store.py

```diff
@@ -1,6 +1,7 @@
-from jinja2 import escape
+from __future__ import annotations
```

Comment:
The current code is not good enough for Rich. He needs the code of the future!!!

Reply:
For anyone wondering, this is to allow using the current class as a type annotation.